### ISSUE TITLE
document_missing_exception error during sync

### ISSUE THREAD

**Issue by GMory** [EOSLaoMao/elasticsearch_plugin#2]
We were able to sync to around 2.5 million blocks, then we started to hit this error. We can attempt to boot the node back up and it seems to continue for a little bit before throwing this again. This was hit on a fresh node using flags `--delete-all-blocks --elastic-index-wipe`.

```
2018-10-04T19:28:40.809 thread-1   elasticsearch_plugin.cpp:505  handle_elasticsearch ] elasticsearch exception, 002748325728f6e67939bd8b26b3b085aadace2bacd59c6eb33c336c30d1658dupdate block {"doc":{"irreversible":true,"validated":true,"in_current_chain":true,"updatedAt":"1538681320801"}}, line 824, 3230005 response_code_exception: Get non 2XX response code from Elasticsearch
404 {"error":{"root_cause":[{"type":"document_missing_exception","reason":"[_doc][002748325728f6e67939bd8b26b3b085aadace2bacd59c6eb33c336c30d1658d]: document missing","index_uuid":"miiN9khZTSGxvdNuBF3NUg","shard":"2","index":"blocks"}],"type":"document_missing_exception","reason":"[_doc][002748325728f6e67939bd8b26b3b085aadace2bacd59c6eb33c336c30d1658d]: document missing","index_uuid":"miiN9khZTSGxvdNuBF3NUg","shard":"2","index":"blocks"},"status":404}    {"code":404,"text":"{\"error\":{\"root_cause\":[{\"type\":\"document_missing_exception\",\"reason\":\"[_doc][002748325728f6e67939bd8b26b3b085aadace2bacd59c6eb33c336c30d1658d]: document missing\",\"index_uuid\":\"miiN9khZTSGxvdNuBF3NUg\",\"shard\":\"2\",\"index\":\"blocks\"}],\"type\":\"document_missing_exception\",\"reason\":\"[_doc][002748325728f6e67939bd8b26b3b085aadace2bacd59c6eb33c336c30d1658d]: document missing\",\"index_uuid\":\"miiN9khZTSGxvdNuBF3NUg\",\"shard\":\"2\",\"index\":\"blocks\"},\"status\":404}"}    thread-1  elasticsearch_client.cpp:112 update

2018-10-04T19:28:41.584 thread-0   net_plugin.cpp:3039           plugin_shutdown      ] shutdown..

2018-10-04T19:28:41.584 thread-0   net_plugin.cpp:3042          plugin_shutdown      ] close acceptor

2018-10-04T19:28:41.584 thread-0   net_plugin.cpp:3045           plugin_shutdown      ] close 110 connections

2018-10-04T19:28:41.587 thread-0   net_plugin.cpp:1363           request_next_chunk   ] Unable to continue syncing at this time

2018-10-04T19:28:41.587 thread-0   net_plugin.cpp:3053           plugin_shutdown      ] exit shutdown

2018-10-04T19:28:41.587 thread-0   elasticsearch_plugin.cpp:226  ~elasticsearch_plugi ] elasticsearch_plugin shutdown in process please be patient this can take a few minutes

2018-10-04T19:28:47.525 thread-1   elasticsearch_plugin.cpp:998  consume_blocks       ] draining queue, size: 5

2018-10-04T19:28:47.551 thread-1   elasticsearch_plugin.cpp:998  consume_blocks       ] draining queue, size: 0

2018-10-04T19:28:47.551 thread-1   elasticsearch_plugin.cpp:1060 consume_blocks       ] elasticsearch_plugin consume thread shutdown gracefully
```

**Comment by ashishchandr70** [comment]
Also, in addition to the log posted by @GMory, here is the error from  **/var/log/elasticsearch/elasticsearch.log** 

[2018-10-04T19:28:40,803][DEBUG][o.e.a.b.TransportShardBulkAction] [blocks][2] failed to execute bulk item (create) BulkShardRequest [[blocks][2]] containing [index {[blocks][_doc][002748325728f6e67939bd8b26b3b085aadace2bacd59c6eb33c336c30d1658d], source[n/a, actual length: [2.8kb], max length: 2kb]}]
org.elasticsearch.index.mapper.MapperParsingException: failed to parse
        at org.elasticsearch.index.mapper.DocumentParser.wrapInMapperParsingException(DocumentParser.java:171) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.index.mapper.DocumentParser.parseDocument(DocumentParser.java:72) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.index.mapper.DocumentMapper.parse(DocumentMapper.java:263) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.index.shard.IndexShard.prepareIndex(IndexShard.java:725) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.index.shard.IndexShard.applyIndexOperation(IndexShard.java:702) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.index.shard.IndexShard.applyIndexOperationOnPrimary(IndexShard.java:682) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.bulk.TransportShardBulkAction.lambda$executeIndexRequestOnPrimary$2(TransportShardBulkAction.java:560) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.bulk.TransportShardBulkAction.executeOnPrimaryWhileHandlingMappingUpdates(TransportShardBulkAction.java:579) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.bulk.TransportShardBulkAction.executeIndexRequestOnPrimary(TransportShardBulkAction.java:558) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.bulk.TransportShardBulkAction.executeIndexRequest(TransportShardBulkAction.java:141) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.bulk.TransportShardBulkAction.executeBulkItemRequest(TransportShardBulkAction.java:247) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.bulk.TransportShardBulkAction.performOnPrimary(TransportShardBulkAction.java:124) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.bulk.TransportShardBulkAction.shardOperationOnPrimary(TransportShardBulkAction.java:111) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.bulk.TransportShardBulkAction.shardOperationOnPrimary(TransportShardBulkAction.java:73) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryShardReference.perform(TransportReplicationAction.java:1017) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryShardReference.perform(TransportReplicationAction.java:995) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.ReplicationOperation.execute(ReplicationOperation.java:101) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction$AsyncPrimaryAction.onResponse(TransportReplicationAction.java:356) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction$AsyncPrimaryAction.onResponse(TransportReplicationAction.java:296) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction$1.onResponse(TransportReplicationAction.java:958) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction$1.onResponse(TransportReplicationAction.java:955) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.index.shard.IndexShardOperationPermits.acquire(IndexShardOperationPermits.java:271) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.index.shard.IndexShardOperationPermits.acquire(IndexShardOperationPermits.java:238) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.index.shard.IndexShard.acquirePrimaryOperationPermit(IndexShard.java:2249) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction.acquirePrimaryShardReference(TransportReplicationAction.java:967) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction.access$500(TransportReplicationAction.java:97) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction$AsyncPrimaryAction.doRun(TransportReplicationAction.java:317) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryOperationTransportHandler.messageReceived(TransportReplicationAction.java:292) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryOperationTransportHandler.messageReceived(TransportReplicationAction.java:279) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.xpack.security.transport.SecurityServerTransportInterceptor$ProfileSecuredRequestHandler$1.doRun(SecurityServerTransportInterceptor.java:251) [x-pack-security-6.4.1.jar:6.4.1]
        at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.xpack.security.transport.SecurityServerTransportInterceptor$ProfileSecuredRequestHandler.messageReceived(SecurityServerTransportInterceptor.java:309) [x-pack-security-6.4.1.jar:6.4.1]
        at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:66) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.transport.TransportService$7.doRun(TransportService.java:665) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:723) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) [elasticsearch-6.4.1.jar:6.4.1]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_141]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_141]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_141]
Caused by: com.fasterxml.jackson.core.JsonParseException: Invalid UTF-8 middle byte 0x22
 at [Source: org.elasticsearch.common.bytes.BytesReference$MarkSupportingStreamInputWrapper@a33ba80; line: 1, column: 1580]
        at com.fasterxml.jackson.core.JsonParser._constructError(JsonParser.java:1702) ~[jackson-core-2.8.10.jar:2.8.10]
        at com.fasterxml.jackson.core.base.ParserMinimalBase._reportError(ParserMinimalBase.java:558) ~[jackson-core-2.8.10.jar:2.8.10]
        at com.fasterxml.jackson.core.json.UTF8StreamJsonParser._reportInvalidOther(UTF8StreamJsonParser.java:3550) ~[jackson-core-2.8.10.jar:2.8.10]
        at com.fasterxml.jackson.core.json.UTF8StreamJsonParser._reportInvalidOther(UTF8StreamJsonParser.java:3557) ~[jackson-core-2.8.10.jar:2.8.10]
        at com.fasterxml.jackson.core.json.UTF8StreamJsonParser._skipUtf8_3(UTF8StreamJsonParser.java:3436) ~[jackson-core-2.8.10.jar:2.8.10]
        at com.fasterxml.jackson.core.json.UTF8StreamJsonParser._skipString(UTF8StreamJsonParser.java:2605) ~[jackson-core-2.8.10.jar:2.8.10]
        at com.fasterxml.jackson.core.json.UTF8StreamJsonParser.nextToken(UTF8StreamJsonParser.java:729) ~[jackson-core-2.8.10.jar:2.8.10]
        at com.fasterxml.jackson.core.base.ParserMinimalBase.skipChildren(ParserMinimalBase.java:154) ~[jackson-core-2.8.10.jar:2.8.10]
        at org.elasticsearch.common.xcontent.json.JsonXContentParser.skipChildren(JsonXContentParser.java:58) ~[elasticsearch-x-content-6.4.1.jar:6.4.1]
        at org.elasticsearch.index.mapper.DocumentParser.parseObjectOrNested(DocumentParser.java:345) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.index.mapper.DocumentParser.parseObjectOrField(DocumentParser.java:478) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.index.mapper.DocumentParser.parseObject(DocumentParser.java:501) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.index.mapper.DocumentParser.innerParseObject(DocumentParser.java:390) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.index.mapper.DocumentParser.parseObjectOrNested(DocumentParser.java:380) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.index.mapper.DocumentParser.internalParseDocument(DocumentParser.java:95) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.index.mapper.DocumentParser.parseDocument(DocumentParser.java:69) ~[elasticsearch-6.4.1.jar:6.4.1]
        ... 38 more
[2018-10-04T19:28:40,806][DEBUG][o.e.a.b.TransportShardBulkAction] [transactions][1] failed to execute bulk item (update) BulkShardRequest [[transactions][1]] containing [update {[transactions][_doc][7dec40712345c3733fb3df80f42fc71ca4c95d0b2f5707bc591908db52a61b10], doc_as_upsert[false], doc[index {[null][null][null], source[{"irreversible":true,"block_id":"002748325728f6e67939bd8b26b3b085aadace2bacd59c6eb33c336c30d1658d","block_num":2574386,"updatedAt":"1538681320801"}]}], scripted_upsert[false], detect_noop[true]}]
org.elasticsearch.index.engine.DocumentMissingException: [_doc][7dec40712345c3733fb3df80f42fc71ca4c95d0b2f5707bc591908db52a61b10]: document missing
        at org.elasticsearch.action.update.UpdateHelper.prepareUpsert(UpdateHelper.java:130) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.update.UpdateHelper.prepare(UpdateHelper.java:86) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.update.UpdateHelper.prepare(UpdateHelper.java:75) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.bulk.TransportShardBulkAction.executeUpdateRequestOnce(TransportShardBulkAction.java:354) ~[elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.bulk.TransportShardBulkAction.executeUpdateRequest(TransportShardBulkAction.java:419) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.bulk.TransportShardBulkAction.executeBulkItemRequest(TransportShardBulkAction.java:251) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.bulk.TransportShardBulkAction.performOnPrimary(TransportShardBulkAction.java:124) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.bulk.TransportShardBulkAction.shardOperationOnPrimary(TransportShardBulkAction.java:111) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.bulk.TransportShardBulkAction.shardOperationOnPrimary(TransportShardBulkAction.java:73) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryShardReference.perform(TransportReplicationAction.java:1017) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryShardReference.perform(TransportReplicationAction.java:995) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.ReplicationOperation.execute(ReplicationOperation.java:101) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction$AsyncPrimaryAction.onResponse(TransportReplicationAction.java:356) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction$AsyncPrimaryAction.onResponse(TransportReplicationAction.java:296) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction$1.onResponse(TransportReplicationAction.java:958) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction$1.onResponse(TransportReplicationAction.java:955) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.index.shard.IndexShardOperationPermits.acquire(IndexShardOperationPermits.java:271) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.index.shard.IndexShardOperationPermits.acquire(IndexShardOperationPermits.java:238) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.index.shard.IndexShard.acquirePrimaryOperationPermit(IndexShard.java:2249) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction.acquirePrimaryShardReference(TransportReplicationAction.java:967) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction.access$500(TransportReplicationAction.java:97) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction$AsyncPrimaryAction.doRun(TransportReplicationAction.java:317) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryOperationTransportHandler.messageReceived(TransportReplicationAction.java:292) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryOperationTransportHandler.messageReceived(TransportReplicationAction.java:279) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.xpack.security.transport.SecurityServerTransportInterceptor$ProfileSecuredRequestHandler$1.doRun(SecurityServerTransportInterceptor.java:251) [x-pack-security-6.4.1.jar:6.4.1]
        at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.xpack.security.transport.SecurityServerTransportInterceptor$ProfileSecuredRequestHandler.messageReceived(SecurityServerTransportInterceptor.java:309) [x-pack-security-6.4.1.jar:6.4.1]
        at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:66) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.transport.TransportService$7.doRun(TransportService.java:665) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:723) [elasticsearch-6.4.1.jar:6.4.1]
        at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37) [elasticsearch-6.4.1.jar:6.4.1]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_141]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_141]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_141]

**Comment by WLBF** [comment]
> We were able to sync to around 2.5 million blocks, then we started to hit this error. We can attempt to boot the node back up and it seems to continue for a little bit before throwing this again. This was hit on a fresh node using flags `--delete-all-blocks --elastic-index-wipe`.
> 
> ```
> 2018-10-04T19:28:40.809 thread-1   elasticsearch_plugin.cpp:505  handle_elasticsearch ] elasticsearch exception, 002748325728f6e67939bd8b26b3b085aadace2bacd59c6eb33c336c30d1658dupdate block {"doc":{"irreversible":true,"validated":true,"in_current_chain":true,"updatedAt":"1538681320801"}}, line 824, 3230005 response_code_exception: Get non 2XX response code from Elasticsearch
> 404 {"error":{"root_cause":[{"type":"document_missing_exception","reason":"[_doc][002748325728f6e67939bd8b26b3b085aadace2bacd59c6eb33c336c30d1658d]: document missing","index_uuid":"miiN9khZTSGxvdNuBF3NUg","shard":"2","index":"blocks"}],"type":"document_missing_exception","reason":"[_doc][002748325728f6e67939bd8b26b3b085aadace2bacd59c6eb33c336c30d1658d]: document missing","index_uuid":"miiN9khZTSGxvdNuBF3NUg","shard":"2","index":"blocks"},"status":404}    {"code":404,"text":"{\"error\":{\"root_cause\":[{\"type\":\"document_missing_exception\",\"reason\":\"[_doc][002748325728f6e67939bd8b26b3b085aadace2bacd59c6eb33c336c30d1658d]: document missing\",\"index_uuid\":\"miiN9khZTSGxvdNuBF3NUg\",\"shard\":\"2\",\"index\":\"blocks\"}],\"type\":\"document_missing_exception\",\"reason\":\"[_doc][002748325728f6e67939bd8b26b3b085aadace2bacd59c6eb33c336c30d1658d]: document missing\",\"index_uuid\":\"miiN9khZTSGxvdNuBF3NUg\",\"shard\":\"2\",\"index\":\"blocks\"},\"status\":404}"}    thread-1  elasticsearch_client.cpp:112 update
> 
> 2018-10-04T19:28:41.584 thread-0   net_plugin.cpp:3039           plugin_shutdown      ] shutdown..
> 
> 2018-10-04T19:28:41.584 thread-0   net_plugin.cpp:3042          plugin_shutdown      ] close acceptor
> 
> 2018-10-04T19:28:41.584 thread-0   net_plugin.cpp:3045           plugin_shutdown      ] close 110 connections
> 
> 2018-10-04T19:28:41.587 thread-0   net_plugin.cpp:1363           request_next_chunk   ] Unable to continue syncing at this time
> 
> 2018-10-04T19:28:41.587 thread-0   net_plugin.cpp:3053           plugin_shutdown      ] exit shutdown
> 
> 2018-10-04T19:28:41.587 thread-0   elasticsearch_plugin.cpp:226  ~elasticsearch_plugi ] elasticsearch_plugin shutdown in process please be patient this can take a few minutes
> 
> 2018-10-04T19:28:47.525 thread-1   elasticsearch_plugin.cpp:998  consume_blocks       ] draining queue, size: 5
> 
> 2018-10-04T19:28:47.551 thread-1   elasticsearch_plugin.cpp:998  consume_blocks       ] draining queue, size: 0
> 
> 2018-10-04T19:28:47.551 thread-1   elasticsearch_plugin.cpp:1060 consume_blocks       ] elasticsearch_plugin consume thread shutdown gracefully
> ```

I try to reperform the problem by replay the block chain, but it does not show up. I find out that the missing index error log is caused by previous docment create failing, so the root cause is not updateing block request. Could you please offer me a full log of nodeos, so I can figure out what cause the first exception. thanks.

**Comment by WLBF** [comment]
It seems that problem is caused by non utf-8 character in memo.

**Comment by WLBF** [comment]
fixed in #3
